powerline/segments/common/time.py

```diff
@@ -3,21 +3,27 @@
 
 from datetime import datetime
 
+from pytz import (timezone, utc)
```

Comment:
It is not technically incorrect. Just you can drop the parenthesis (leaving everything else as-is) and according to the current style you should drop it unless you are going to have multiline import.